### ISSUE TITLE
Controller does not turn off

### ISSUE THREAD

**Issue by pdien** [ublue-os/bazzite#1289]
### Describe the bug

Whether in Game Mode or the desktop, the connected controller will not go to sleep. Auto-sleep function is set to 15 minutes and it still stays on past that time, draining the battery. HOME + △/Y also does not turn the controller off.

### What did you expect to happen?

Controller should be turning off.

### Output of `rpm-ostree status`

```shell
State: idle
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:stable
                   Digest: sha256:b48da348e0f942a18e833fb14f5483d70b291b41f34089c390148ad53a36dec4
                  Version: 40.20240625.0 (2024-06-27T02:01:04Z)
          LayeredPackages: sunshine

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:stable
                   Digest: sha256:b48da348e0f942a18e833fb14f5483d70b291b41f34089c390148ad53a36dec4
                  Version: 40.20240625.0 (2024-06-27T02:01:04Z)
          LayeredPackages: sunshine
```

### Hardware

![image](https://github.com/ublue-os/bazzite/assets/22202244/4452735b-2626-4933-a3f6-57506b4300d5)

### Extra information or context

Controllers used: DualSense, Dualshock 4, Xbox One, Xbox Series X.
Turning off bluetooth does turn them off, but this is not the ideal use case.

**Comment by thekk1** [comment]
The workaround #1390 will not handle this issue.
Especial not the part with the shortcut HOME + △/Y.

**Comment by thekk1** [comment]
I can confirm this behavior on my machine UM790pro.

**Comment by pdien** [comment]
This might be a general Linux problem, as suggested by S1ngl3-x in #1040. I installed a fresh Ubuntu image, installed Steam, connected a DualSense controller and the HOME + △/Y shortcut still doesn't work.

**Comment by thekk1** [comment]
This issue is tracked at steam under [#8678](https://github.com/ValveSoftware/steam-for-linux/issues/8678)

**Comment by thekk1** [comment]
Who does not want to wait for the fix #1416 can try the script from here:
https://github.com/thekk1/bluetooth-ds4-ds5-workaround
